README.md

```diff
@@ -312,6 +344,9 @@ crud.select('customers', {{'<=', 'age', 35}})
 ...
 ```
 
+**Note**: that tuples are sorted by age because space has index `age`.
```

Comment:
I think "that" (after `:`) isn't needed here. Check please

Reply:
Fixed.